src/fava/templates/macros/_commodity_macros.html

```diff
@@ -0,0 +1,20 @@
+{% macro render_currency(currency) %}
```

Comment:
I think I'd prefer it if this (and the macro below) would take an explicit `ledger` first argument to make the required context a bit clearer (and allowing this file to be imported without context).

Reply:
The `ledger` variable has been used a lot in templates so I think it makes sense to [leave it as a global variable](https://github.com/beancount/fava/blob/a8c4f03c1a3b2a5c452858b71b84887782e91602/src/fava/application.py#L205). Otherwise I have to pass the variable as a parameter in many macro calls.